### PR TITLE
Add plots aimec

### PR DESCRIPTION
**Pull request by matthiasto**
Fixed entrainment bug in com1DFAPy (we were always using entrainment...)
Added comparison plot to AIMEC

### CODE REVIEW

File: avaframe/out3Plot/outAIMEC.py [hunk 1]
```diff
@@ -117,13 +117,92 @@ def visuRunout(rasterTransfo, resAnalysis, plim, newRasters, cfgPath, cfgFlags):
...
+    ax3.plot(fvCrossMaxAll[1, :], s, '--b', label='Max Simulation')
+    ax3.plot(fvCrossMeanAll[1, :], s, '-b', label='Mean Simulation')
+
+    ax3.set_title('Flow velocity distribution along the path')
+    ax3.legend(loc=4)
+    ax3.set_ylabel('l [m]')
+    ax3.set_ylim([s.min(), s.max()])
+    ax3.set_xlim(auto=True)
+    ax3.set_xlabel('$v(s) [m.s^{-1}]$')
+
```
Comment (fso42) [inline avaframe/out3Plot/outAIMEC.py:171]: I suggest making a small internal function that does the plotting and styling which you can call 3 time with the input
`XXCrossMaxAll, label, title`

**Review by fso42 — CHANGES_REQUESTED** [review]
Since there is a lot of overlap with PR #342 I only looked ad the outAIMEC.py. Once that PR is merged, rebase on the master and then I'll review it again if needed...

**Review by fso42 — APPROVED** [review]

**Comment by matthiasto** [comment]
replaced by branch refactorAimec #346

**Review by awirb — COMMENTED** [review]
* suggestion: add a warning if aimec doesn't start to run, e.g. if no results from one computational module are available

**Pull request closed without merging**